tests/director/external-services-mock-integration/async_api_delete_test.go

```diff
@@ -0,0 +1,171 @@
+package external_services_mock_integration
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/kyma-incubator/compass/components/operations-controller/api/v1alpha1"
+	"github.com/kyma-incubator/compass/components/operations-controller/client"
+	"github.com/kyma-incubator/compass/tests/director/pkg/gql"
+	"github.com/kyma-incubator/compass/tests/director/pkg/idtokenprovider"
+	gcli "github.com/machinebox/graphql"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
+	"github.com/kyma-incubator/compass/components/director/pkg/str"
+	"github.com/kyma-incubator/compass/components/external-services-mock/pkg/webhook"
+	"github.com/kyma-incubator/compass/tests/director/pkg/ptr"
+)
+
+func TestAsyncAPIDeleteApplication(t *testing.T) {
+	var (
+		operationFullPath = fmt.Sprintf("%s%s", testConfig.ExternalServicesMockBaseURL, webhook.OperationPath)
+		deleteFullPath    = fmt.Sprintf("%s%s", testConfig.ExternalServicesMockBaseURL, webhook.DeletePath)
+	)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	appName := fmt.Sprintf("app-async-del-%s", time.Now().Format("060102150405"))
```

Comment:
You can use ``time.Now().Format(time.RFC3339Nano)` here instead of hardcoding.
Example: https://github.com/kyma-incubator/compass/blob/252d77cd7a5028dc39e6d6b8f782b7e58ad07bc6/components/operations-controller/internal/k8s/status/manager_test.go#L252

Reply:
time.RFC3339Nano  contains special symbols, spaces and is too long for this field.